ssl-params.conf

```diff
@@ -0,0 +1,18 @@
+ssl_protocols TLSv1.2;
+ssl_prefer_server_ciphers on;
+ssl_dhparam /etc/nginx/dhparam.pem;
+ssl_ciphers ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384;
```

Comment:
if you were to chose one cipher, which one would you use?

Reply:
RSA-AES256 better against brute force attacks
RSA-AES128 faster and more efficent
Anyways 128 or 256bits  algorithms are pretty strong

Reply:
ok